src/drive/web/modules/drive/Toolbar/components/Grid/GridItem.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import Grid from 'cozy-ui/react/MuiCozyTheme/Grid'
+
+class GridItem extends Component {
+  render() {
+    const { onClick, children } = this.props
+    return (
+      <Grid
+        item
+        xs={3}
+        onClick={() => onClick && onClick()}
```

Comment:
We could give `onClick` a default value of `() => {}` and then we don't need the check here?